app/experimenter/nimbus-ui/src/components/Summary/EndExperiment/index.tsx

```diff
@@ -12,55 +12,19 @@ const EndExperiment = ({
   isLoading: boolean;
   onSubmit: () => void;
 }) => {
-  const [showEndConfirmation, setShowEndConfirmation] = useState(false);
-
-  const toggleShowEndConfirmation = useCallback(
-    () => setShowEndConfirmation(!showEndConfirmation),
-    [showEndConfirmation, setShowEndConfirmation],
-  );
-
   return (
     <div className="mb-4" data-testid="experiment-end">
-      {showEndConfirmation ? (
-        <Alert variant="secondary" data-testid="end-experiment-alert">
-          <p>
-            Are you sure you want to end your experiment? It will turn off the
-            experiment for all users in production.
-          </p>
-
-          <div>
-            <Button
-              variant="primary"
-              onClick={onSubmit}
-              className="end-experiment-confirm-button"
-              disabled={isLoading}
-              data-testid="end-experiment-confirm"
-            >
-              Yes, end the experiment
-            </Button>
```

Comment:
The integration tests are failing because they are looking for this button. I think you can just remove these lines, https://github.com/mozilla/experimenter/blob/main/app/tests/integration/nimbus/pages/experimenter/summary.py#L149-#L154